about/about-kubernetes-ingress.md

```diff
@@ -0,0 +1,130 @@
+HTTPS requests. The requests are then forwarded via separate connections from the load balancer to the chosen service
+backing pods. As a result, network policy applied to the backing pods can restrict access to only allow connections from the load
+balancer, but cannot restrict access to specific original clients.
+
+### Why use Kubernetes Ingress?
+
+Given that Kubernetes [Services]({{site.baseurl}}/about/about-kubernetes-services) already provide a mechanism for load
+balancing access to services from outside of the cluster, why might you want to use Kubernetes Ingress?
+
+The mainline use case is if you have multiple HTTP / HTTPS services which you want to expose through a single external IP
+address, perhaps with each service having a different URL path, or perhaps as multiple different domains. This can be a
+lot simpler from a client configuration point of view compared to exposing each service outside of the cluster using
+Kubernetes Services which gives each service it's own external IP address.
+
+If on the other hand, your application architecture is fronted by a single "front end" microservice then Kubernetes
+Services likely already meet your needs and you might prefer to not add Ingress to the picture, both from a simplicity
+point of view, and potentially also so you can more easily restrict access to specific clients using network policy.
+In effect, your "front end" microservice already plays the role of Kubernetes Ingress, not that dissimilar to
+[in-cluster ingress](#in-cluster-ingress-solutions) solutions discussed below.
+
+### Types of Ingress solutions
+
+Broadly speaking there are two types of ingress solutions:
+- In-cluster ingress - where the ingress load balancing is performed by pods within the cluster itself.
+- External ingress - where the ingress load balancing is implemented outside of the cluster by
+  appliances or cloud provider capabilities.
+
+#### In-cluster ingress solutions
+
+In-cluster ingress solutions use software load balancers running in pods within the cluster itself. There are many
+different ingress controllers to consider that follow this pattern, including for example the NGINX ingress controller.
+
+The advantages of this approach are that you can: 
+- horizontally scale your ingress solution up to the limits of Kubernetes
+- choose the ingress controller that best suits your specific needs, for example, with particular load balancing
+  algorithms, or security options.
+
+To get your ingress traffic to the in-cluster ingress pods, the ingress pods are normally exposed externally as a
+Kubernetes service, and any of the standard ways of accessing the service from outside of the cluster can be used. A
+common approach is use an external network load balancer, or service IP advertisement, with
+`externalTrafficPolicy:local` to minimize the number of network hops, and to retain client source IP address, which
```

Comment:
A common approach is to use an external network load balancer, or service IP advertisement so you can use network policy to restrict access to the ingress pods to particular clients using `externalTrafficPolicy:local` to minimize the number of network hops, and retain client source IP address.

Reply:
I think I'll leave this one as is. externalTrafficPolicy local is a property of the service, so should be next to the other properties of the service I think.

Reply:
I have split the sentence into two so it easier to parse though!